src/content/quickstart/verify_with_discovery.md

```diff
@@ -16,9 +16,25 @@ kubectl -n default  run --generator=run-pod/v1 tmp-shell --rm -i --tty --image q
 curl nginx.default.svc.supercluster.local:8080
 ```
 
+To verify a headless service follow the steps below using a mehdb service
+
+```bash
+export KUBECONFIG=cluster-b/auth/kubeconfig
+kubectl apply -f https://raw.githubusercontent.com/openshift-evangelists/mehdb/master/app.yaml
+subctl export service --namespace default mehdb
+```
+
+```bash
+export KUBECONFIG=cluster-a/auth/kubeconfig
+kubectl -n default  run --generator=run-pod/v1 tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
+dig mehdb.default.svc.clusterset.local:8080
+```
+
+The dig will return a set of A records containing the IP address of Pods backing the service mehdb in cluster-b.
```

Comment:
Good to add sample output here as default output can be too verbose and user not sure what to look for. Alternately, use `+short`

Reply:
added +short